src/qt/transactionview.cpp

```diff
@@ -355,7 +355,11 @@ void TransactionView::exportClicked()
     writer.addColumn(tr("Type"), TransactionTableModel::Type, Qt::EditRole);
     writer.addColumn(tr("Label"), 0, TransactionTableModel::LabelRole);
     writer.addColumn(tr("Address"), 0, TransactionTableModel::AddressRole);
-    writer.addColumn(BitcoinUnits::getAmountColumnTitle(model->getOptionsModel()->getDisplayUnit()), 0, TransactionTableModel::FormattedAmountRole);
+    if (model && model->getOptionsModel()) {
```

Comment:
In case `model` or `model->getOptionsModel()` is not set, there's no reason to continue here - there's no data to export.
Might as well return from the function early if so.